src/settings.js

```diff
@@ -326,6 +326,9 @@ var SOCKET_WEBRTC = 0;
 // where addr and port are derived from the socket connect/bind/accept calls.
 var WEBSOCKET_URL = 'ws://';
 
+// If 1, the POSIX sockets API uses a proxy bridge to proxy sockets calls
```

Comment:
please expand on this, since "proxying" can mean "proxy from a pthread to the main thread" but I think here it means something very different?

Reply:
I think it's unfortunate ambiguity created by using the appropriate word for similar things in different places. Maybe we could agree on different nomenclature to avoid the ambiguity, such as POSIX sockets _bridge_?

Reply:
ok, updated